src/pybind/mgr/restful/api/config.py

```diff
@@ -33,7 +33,7 @@ def patch(self, **kwargs):
         valid_flags = set(args.keys()) & set(common.OSD_FLAGS)
         invalid_flags = list(set(args.keys()) - valid_flags)
         if invalid_flags:
-            context.instance.log.warn("%s not valid to set/unset" % invalid_flags)
+            context.instance.log.warn("{} not valid to set/unset".format(invalid_flags))
```

Comment:
probably a more pythonic change is to put
```py
context.instance.log.warn("%s not valid to set/unset", invalid_flags)
```

Reply:
@tchaikov I can make that change. As a test, I created a list (like the ``OSD_FLAGS`` list in ``common.py``):

``flags = ['osdflag1']``

If I modify the code like you recommended, would I get this as a return, being that someone threw an ``OSD_FLAG`` that was not valid?

``%s not valid to set/unset ['osdflag1']``

Reply:
@falcon78921 please see https://docs.python.org/3/library/logging.html#logging.debug for the answer.

Reply:
@tchaikov Thank you for the explanation Kefu. I made the change.